### PR TITLE
Welcome chat message

### LINKED ISSUE #142: New Player Welcoming Features:  Chat Message on Arrival/Exit Tutorial
Chat message emit by System(?) on Help, General(?) .. misc.. chat channels when a new player, character (first character only?) arrives and/or exits the tutorial.

### PR DESCRIPTION
**Pull request by clouths**
Closes: [OpenPerpetuum/OP-Project#142](https://github.com/OpenPerpetuum/OP-Project/issues/142)
For: [OpenPerpetuum/PerpetuumServer#100](https://github.com/OpenPerpetuum/PerpetuumServer/pull/100)

The test procedure is located [here](https://github.com/OpenPerpetuum/PerpetuumServer/pull/100)

### CODE REVIEW

File: Staging_Dev/20190508T014400Z_welcome_chat_message.sql [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+	id int primary key,
+	name varchar(32) not null unique,
+	message nvarchar(2000) not null
+);
+GO
+
+
+
+INSERT INTO dbo.premadechatmessage (id, name, message)
+VALUES (1, 'TutorialCompleteMessage', 'Attention Nia members, we have a new challenger to the world: $NAME$');
```
Comment (MikeJeffers) [inline Staging_Dev/20190508T014400Z_welcome_chat_message.sql]: Let's workshop the message content.

File: Staging_Dev/20190508T014400Z_welcome_chat_message.sql [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+           ,[globalMute]
+           ,[avatar]
+           ,[note]
+           ,[corporationeid]
+           ,[allianceeid]
+           ,[language])
+VALUES
+			((SELECT accountID FROM dbo.accounts where email = 'OPP_server_account'), --accountID
+			8702057415139945528, 
+			N'New Player Announcer', --nick
```
Comment (MikeJeffers) [inline Staging_Dev/20190508T014400Z_welcome_chat_message.sql]: And might consider a new name (prepend "[OPP]" ?) Will discuss

**Review by MikeJeffers — CHANGES_REQUESTED** [review]
Some content tweaks

**Review by MikeJeffers — APPROVED** [review]

**Comment by MikeJeffers** [comment]
@Bler Still needs your signoff - merge when ready

**Review by Bler — APPROVED** [review]
Approved

**Pull request merged**